### PR TITLE
Fix create account form styling

### PR DESCRIPTION
**Pull request by rekmarks**
There was no space between the buttons in the create account form. This has been fixed.

<details>
<summary>Before</summary>

</details>

<details>
<summary>After</summary>

</details>

### CODE REVIEW

**Review by brad-decker — DISMISSED** [review]

File: ui/app/pages/create-account/new-account.component.js [hunk 1]
```diff
@@ -49,31 +49,31 @@ export default class NewAccountCreateForm extends Component {
         <div className="new-account-create-form__input-label">
           {this.context.t('accountName')}
         </div>
-        <div className="new-account-create-form__input-wrapper">
+        <div className="new-account-create-form__account-name">
```
Comment (Gudahtt) [inline ui/app/pages/create-account/new-account.component.js:52]: Does this class exist? I can't find it

Comment (Gudahtt) [reply ui/app/pages/create-account/new-account.component.js:52]: I can't find the old one either 😕

Comment (rekmarks) [reply ui/app/pages/create-account/new-account.component.js:52]: Neither the old nor the new class exist, but the div wrapping of both the input and the button solves the problem. Delete the class entirely?

Comment (Gudahtt) [reply ui/app/pages/create-account/new-account.component.js:52]: >Delete the class entirely?

Sure, that sounds like it would be best

**Review by Gudahtt — APPROVED** [review]
LGTM!

**Pull request merged**